unlock-js/src/v10/createLock.js

```diff
@@ -16,6 +17,18 @@ export default async function(lock, owner) {
   }
   let currencyContractAddress = lock.currencyContractAddress || ZERO
 
+  let decimalKeyPrice
```

Comment:
Let's move this bit of logic into its own method/module

Reply:
Done! Thanks, PTAL!